src/main/java/com/coveo/AbstractFMT.java

```diff
@@ -145,11 +154,8 @@ private void formatSourceFilesInDirectory(File directory) throws MojoFailureExce
     throw new MojoFailureException(message);
   }
 
-  private synchronized Formatter formatter() throws MojoFailureException {
-    if (formatter == null) {
-      formatter = new Formatter(JavaFormatterOptions.builder().style(style()).build());
-    }
-    return formatter;
+  private Formatter getFormatter() throws MojoFailureException {
```

Comment:
There's a change of behavior here, not sure how relevant it is:

It used to be that the formatter wouldn't be initialized if there was no file to format.
If we don't care, we should move the init of the formatter in the constructor.

Reply:
Something tells me that it was lazy initialized for a reason 🤔

Reply:
From what I understand, it was done lazily to allow the formatter to take into account the JavaFormatterOption.Style that is now taken as a parameter (can be "oasp" or "google").

By providing the formatter as a parameter, I keep the same functionnality (initialize the Formatter only once the style is set) while removing the need for a synchronized getter.